mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+
+    Returns
+    -------
+    intervals : pandas.DataFrame
+        All the data from the feature directory into one data frame
+        columns are strain, mouse, day, start, stop
+    """
+    # get all file names
```

Comment:
Sure [this should help](http://stackoverflow.com/questions/16021096/python-best-way-to-check-if-list-is-empty-without-not)

The simple way is to use the `if not file_names:` syntax to check for emptiness. There is a subtle issue that this also picks up `None` objects. However we can check with Jarrod if and how to  best capture these.